.travis.yml

```diff
@@ -14,6 +14,7 @@ script:
   - ./scripts/check-pristine-state package-lock.json
   - npm run danger
   - npm test
+  - npm run submodules:pull
```

Comment:
This change means we are giving up the ability to control which changes we push to the docs.

Previously we were 100% in control (perhaps at a burden). I want to see others opinions on this.

Reply:
My only problem with this is we run the risk of testing locally with an outdated version and maybe find a problem only after it is published with the latest version of the submodules. At the moment we always test with the latest version locally and this ensures we don't run into this problem

Reply:
@matthewdavidson previously travis would have always pulled the latest version anyway...